.github/CODEOWNERS

```diff
@@ -1 +1 @@
-*	@loadsmart/mobile
+*	@loadsmart/roadrunners @loadsmart/mobile
```

Comment:
@roquebetioljr same case here as [dangerfile](https://github.com/loadsmart/dangerfile). Also, I think the documentation check will block the PR merge.

Reply:
I'm checking it with platform.